daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -169,9 +180,13 @@ object Speedy {
       }
     }
 
-    @inline def restoreEnv(frameToBeRestored: Frame, envSize: Int): Unit = {
-      // Restore the frame pointer captured when the continuation was created.
+    @inline def restoreEnv(
+        frameToBeRestored: Frame,
+        actualsToBeRestored: Actuals,
+        envSize: Int): Unit = {
```

Comment:
I think I already mentioned this earlier but I find `envSize` a very undescriptive name. Shouldn't it be `envSizeToBeRestored` instead?

Reply:
ok, this is more descriptive. I changed it.